schema-util/util-provider/src/main/java/io/apicurio/registry/types/provider/AvroArtifactTypeUtilProvider.java

```diff
@@ -37,22 +36,19 @@ private static String removeQuotedBrackets(String content) {
     @Override
     public boolean acceptsContent(TypedContent content, Map<String, TypedContent> resolvedReferences) {
         try {
-            String contentType = content.getContentType();
```

Comment:
@EricWittmann I had to remove this condition, otherwise, when the contentType is null (like in the v3 export case), this does not work. I guess this condition was here as an optimization or there was some case where it was actually protecting from a potential error?

Reply:
Right.  Can we add something like this back in?

```
if (content.getContentType() != null && content.getContentType().toLowerCase().contains("json") && !ContentTypeUtil.isParsableJson(content.getContent())) {
    return false;
}
```

wdyt?

Reply:
The idea is that we don't attempt the (expensive?) parsing operations unless we have reason to think it might work (the content-type matches something we think should work).